audio/softsynth/fmtowns_pc98/towns_pc98_driver.cpp

```diff
@@ -213,8 +213,8 @@ _sustain(false), _fading(false), _dataPtr(nullptr), _vbrModInitVal(0), _vbrModCu
 #undef CONTROL
 
 TownsPC98_MusicChannel::~TownsPC98_MusicChannel() {
-	for (Common::Array<const ControlEvent*>::iterator i = _controlEvents.begin(); i != _controlEvents.end(); ++i)
-		delete *i;
+	for (auto &event : _controlEvents)
+		delete event;
```

Comment:
Iterators to pointers are a little interesting in this conversion. I think as written these are now references to pointers, which might be fine. Would there be clarity or benefit to "auto *" for those?

Reply:
Hm, yes, you are right that deleting references would be clearer if we use `auto *` in these cases. Will adapt these cases accordingly

Reply:
@OMGPizzaGuy done. Thanks for your feedback once again! :)